src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -244,13 +260,41 @@ private void writeToFinalFile(JsonNode jsonNode, String name, String controllerN
 		}
 		
 		
-		
+		result.append("\n}");
 		input = result.toString();
-//		input = input.replace("null", "");
+	
 		
-		input = input.replace("{1}", controllerName);
-		input = input.replace("ControllerName", controllerName);
-		writeToJavaFile(input, name);
+//		input = input.replace("ControllerName", controllerName);
+//		if(fileIndex == 0)
+//		{
+//			writeToJavaFile(input, name, "D:\\restapi\\nextgen-sti\\src\\main\\java\\pb360\\model");
```

Comment:
don't use hard code, please remove. please research about use constant for static variable in java